server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -1005,7 +1024,7 @@ public void test_add_geo_shape_column() throws Exception {
                    "id" INTEGER,
                    "g" GEO_SHAPE GENERATED ALWAYS AS 'POLYGON (( 5 5, 30 5, 30 30, 5 30, 5 5 ))' INDEX USING QUADTREE WITH (
                       distance_error_pct = 0.123,
-                      precision = '123.0m'
+                      precision = '123m'
```

Comment:
Just out of interest, what caused this change?

Reply:
I haven't double checked but I suspect the geo field mappers normalized the representation